src/DoughnutChart.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+class DoughnutChart extends PieChart
+{
+    /** @var string Type of chart */
```

Comment:
never copy phpdoc, if overriden

also override properties should be listed first (here fine, just to understand the CS rules)